<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>feedbackDialog</class>
 <widget class="QDialog" name="feedbackDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>460</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="messBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>731</width>
     <height>231</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="messEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>731</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>举报与反馈</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>编辑您的内容：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
